### PR TITLE
US01-04: Création d'une caroussel pour différentes page

### PR DESCRIPTION
**Pull request by Wowlfy**

### CODE REVIEW

File: public/assets/css/carousel.css [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    }
+    99% {
+        scroll-snap-align: none;
+    }
+    100% {
+        scroll-snap-align: center;
+    }
+}
+
+body {
```
Comment (bouboumael) [inline public/assets/css/carousel.css]: global element if you define it like this, all element identical to the page will be broken by your css integration

File: public/assets/css/carousel.css [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+    max-width: 37.5rem;
+    margin: 0 auto;
+    padding: 0 1.25rem;
+    font-family: 'Lato', sans-serif;
+}
+
+* {
+    box-sizing: border-box;
+    scrollbar-color: transparent transparent;
```
Comment (bouboumael) [inline public/assets/css/carousel.css]: no scrollbar in your page ?

File: public/assets/css/carousel.css [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+    background: transparent;
+    border: none;
+}
+
+* {
+    -ms-overflow-style: none;
+}
+
+ol,
+li {
```
Comment (bouboumael) [inline public/assets/css/carousel.css]: redifined a class for target global element

**Review by bouboumael — CHANGES_REQUESTED** [review]

File: public/assets/css/carousel.css (hunk 1 above)
Comment (Wowlfy) [reply public/assets/css/carousel.css]: okay

File: public/assets/css/carousel.css (hunk 3 above)
Comment (Wowlfy) [reply public/assets/css/carousel.css]: okay

File: public/assets/css/carousel.css (hunk 2 above)
Comment (Wowlfy) [reply public/assets/css/carousel.css]: okay

**Comment by Wowlfy** [comment]
okay

**Review by ChristianOlmedo — CHANGES_REQUESTED** [review]
I think you shouldn't had to delete db.php.dist

File: src/View/User/Includes/carousel.html.twig [hunk 4]
```diff
@@ -0,0 +1,21 @@
+<div id="carouselExampleControls" class="carousel slide" data-bs-ride="carousel">
+  <div class="carousel-inner">
+    <div class="carousel-item active">
+      <img src="/assets/images/accueil.jpg" class="d-block w-100" style="max-width: auto; height: auto;">
```
Comment (bouboumael) [inline src/View/User/Includes/carousel.html.twig]: remove the style in the tag and try to make the image sizes identical

**Review by bouboumael — CHANGES_REQUESTED** [review]

**Review by bouboumael — APPROVED** [review]

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
16 FILES CHANGED? + changes to composer.json and composer.lock?? and to db.php.dist?? + what are those .DS_Store files??

**Pull request closed without merging**